beacon-chain/rpc/prysm/v1alpha1/validator/aggregator_test.go

```diff
@@ -29,6 +30,13 @@ import (
 	"github.com/prysmaticlabs/prysm/v5/testing/util"
 )
 
+// pubKey is a helper to generate a well-formed public key.
+func pubKey(i uint64) []byte {
```

Comment:
This was originally defined in `beacon-chain/rpc/prysm/v1alpha1/validator/duties_test.go`, but since we moved that to a minimal test, it needs to be redefined and can no longer be shared.